chat-components/src/services/BroadcastService.ts

```diff
@@ -2,16 +2,40 @@ import { Subject } from "rxjs";
 import { ICustomEvent } from "../interfaces/ICustomEvent";
 import { filter } from "rxjs/operators";
 import { BroadcastChannel } from "broadcast-channel";
-import { BROADCAST_CHANNEL_NAME } from "../common/Constants";
 
 const newMessage = new Subject<ICustomEvent>();
 
-const pubChannel = new BroadcastChannel(BROADCAST_CHANNEL_NAME);
-const subChannel = new BroadcastChannel(BROADCAST_CHANNEL_NAME);
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const broadcastServicePubList: Record<string, any> = {};
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const broadcastServiceSubList: Record<string, any> = {};
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-subChannel.onmessage = (message: any) => {
-    newMessage.next(message);
+let pubChannel: any;
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+let subChannel: any;
+
+export const BroadcastServiceInitialize = (channelName: string) => {
```

Comment:
Please check if all lcw events are still being raised from script layer after this change

Reply:
okay, will validate them before raising widget PR